<?xml version="1.0" ?>
<launch>

  <arg name="suction_tools"	default="$(dirname
					 )/../config/suction_tools.yaml"/>
  <arg name="operation_mode"	default="$(dirname
					 )/../config/operation_mode.yaml"/>

  <node name="suction_tool_controller"
	pkg="aist_fastening_tools" type="suction_tool_controller.py"
	output="screen">
    <rosparam param="suction_tools"	file="$(arg suction_tools)"/>
    <rosparam param="operation_mode"	file="$(arg operation_mode)"/>
  </node>

</launch>
